client/src/components/navbar/UpdateCourses.js

```diff
@@ -0,0 +1,71 @@
+/** @jsx jsx */
+
+import {css, jsx} from "@emotion/core";
+import {getToken} from "../../utils/authService";
+
+function UpdateCourses() {
+
+  const style = css`
+
+    & {
+      display: inline-block;
+      height: 35px;
+    }
+
+    update-courses-button {
+      height: 35px;
+      border: 1px solid white;
+      color: white;
+      border-radius: 0.25rem;
+      background: transparent;
+    }
+
+  `;
+
+  // update the available courses in the database
+  // using the OSU course catalog
+  async function startCourseUpdate() {
+
+    const firstPrompt = "Update the list of available courses to match the " +
+      "current OSU course catalog?";
+    const secondPrompt = "Updating the courses may take anywhere from a few minutes " +
+      "to a few hours.\n\n " +
+      "This action should only be performed when there is not " +
+      "a large amount of user traffic and only when there is the expectation " +
+      "of new courses existing in the OSU course catalog (once a term).\n\n" +
+      "Do you still want to update the current list of available courses?";
+
+    if (window.confirm(firstPrompt) && window.confirm(secondPrompt)) {
+
+      try {
+
+        // request that all courses are updated
+        const token = getToken();
+        const server = `${process.env.REACT_APP_API_HOST}:${process.env.REACT_APP_API_PORT}`;
+        const url = `http://${server}/course/updateDatabase?accessToken=${token}`;
+
+        // perform the query
+        const response = await fetch(url);
+        if (!response.ok) {
+          // note that there was an error trying to start the update
+          alert("Internal server error. Unable to update courses.");
+        }
+
+      } catch (err) {
+        // note that there was an error trying to start the update
+        alert("Internal server error. Unable to update courses.");
+      }
+    }
+
+  }
+
+  return (
+    <div css={style}>
+      <button className="update-courses-button" onClick={() => startCourseUpdate()}>
+        Update Courses
+      </button>
+    </div>
```

Comment:
![image](https://user-images.githubusercontent.com/14208546/78198983-3938b200-743f-11ea-9167-ce41377b82c6.png)

For some reasons, I couldn't see the `UpdateCourses` button even though I was logged in as a Head Advisor. The DOM looks good to me, but I still couldn't see the button.

The console didn't throw any error.

Reply:
I'm dumb. I opened the up-to-date version of your branch remotely on WSL and ran the out-of-date version on my Windows system. It's working.